src/types/streams/streams_item.rs

```diff
@@ -1,17 +1,16 @@
-use crate::types::resource::Stream;
 use chrono::{DateTime, Utc};
 use serde::{Deserialize, Serialize};
 use serde_with::serde_as;
-use url::Url;
+
+use crate::types::{addon::ResourceRequest, resource::Stream};
 
 #[serde_as]
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct StreamsItem {
     pub stream: Stream,
-    pub meta_id: String,
-    pub video_id: String,
```

Comment:
We should keep `meta_id` and `video_id` because sometimes it's easier to iterate through values instead of using the `StreamsItemKey`

Reply:
Comment is resolved. I've merged your PR #523